### DISCUSSION TITLE
Workflow name to be exposed in environment variables

### DISCUSSION THREAD

**Discussion by Reedyuk** [codemagic-ci-cd/codemagic-docs#1799] [Ideas]
### Use case

```Text
I have a slack script which outputs when a build is successful or not. It would be nice if I was able to get the workflow name that failed or succeeded for the slack message
```

### Idea

```Text
Pretty much the usecase of above. 
For now, i could call the api, but makes more sense to have this as an environment variable

Must be the human readable name e.g. Pull Request Workflow, Release Workflow
```

**Comment by Ycoi** [comment]
Thanks for the idea! We discussed this in the team and this should be rather straightforward to implement and indeed a nice addition. I’ll let you know when we know more about when this could be available.

**Comment by Ycoi** [comment]
@Reedyuk This is now added. The workflow name is available via the `CM_WORKFLOW_NAME` environment variable. This is also updated in our [docs](https://docs.codemagic.io/yaml-basic-configuration/environment-variables/)

Test it out and let me know here if you encounter any issues.

**Comment by Reedyuk** [comment]
wow super quick, thanks!
